src/enums/nr-state.ts

```diff
@@ -6,6 +6,7 @@ export enum NrState {
   COMPLETED = 'COMPLETED', // NOT A VALID STATE
   COND_RESERVED = 'COND-RESERVE', // INTERIM STATE
   CONDITIONAL = 'CONDITIONAL',
+  CONSUMED = 'CONSUMED',
```

Comment:
Is this a state that exists in Namex?

Reply:
Yes. PR for that in review

Reply:
Very cool, this will be useful in other places as well once Consumed is an official state 👍

Reply:
What about Name-Examination UI?

Reply:
I am working on that now